x/token/types/params.go

```diff
@@ -14,26 +15,33 @@ const (
 	DefaultFeeMint   = "10"
 	DefaultFeeBurn   = "10"
 	DefaultFeeModify = "0"
-	DefaultFeeChown     = "10"
+	DefaultFeeChown  = "10"
 )
 
 var (
-	KeyFeeIssue     = []byte("FeeIssue")
-	KeyFeeMint      = []byte("FeeMint")
-	KeyFeeBurn      = []byte("FeeBurn")
-	KeyFeeModify    = []byte("FeeModify")
-	KeyFeeChown     = []byte("FeeChown")
+	KeyFeeIssue                       = []byte("FeeIssue")
+	KeyFeeMint                        = []byte("FeeMint")
+	KeyFeeBurn                        = []byte("FeeBurn")
+	KeyFeeModify                      = []byte("FeeModify")
+	KeyFeeChown                       = []byte("FeeChown")
+	KeyCertifiedTokenMinDeposit       = []byte("CertifiedTokenMinDeposit")
+	KeyCertifiedTokenMaxDepositPeriod = []byte("CertifiedTokenMaxDepositPeriod")
+	KeyCertifiedTokenVotingPeriod     = []byte("CertifiedTokenVotingPeriod")
 )
 
 var _ params.ParamSet = &Params{}
 
 // mint parameters
 type Params struct {
-	FeeIssue     sdk.DecCoin `json:"issue_fee"`
-	FeeMint      sdk.DecCoin `json:"mint_fee"`
-	FeeBurn      sdk.DecCoin `json:"burn_fee"`
-	FeeModify    sdk.DecCoin `json:"modify_fee"`
-	FeeChown     sdk.DecCoin `json:"transfer_ownership_fee"`
+	FeeIssue  sdk.DecCoin `json:"issue_fee"`
```

Comment:
the json contents are not equal to the name of param.
it is best to write like this:
```
FeeIssue  sdk.DecCoin `json:"fee_issue"`
```